packages/css-library/dist/tokens/css/variables.css

```diff
@@ -11,7 +11,7 @@
   --small-desktop-screen: 1008px;
   --large-screen: 1201px;
   --color-link-default-hover: rgba(#000000, 0.05);
-  --vads-button-color-background-primary-alt-active-on-light: #154c21;
+  --vads-button-color-background-primary-alt-active-on-light: #0a200f;
```

Comment:
Is this hex color supposed to change?

Reply:
huh, that is weird. Where is that coming from :thinking:

Reply:
I rebuilt the tokens in dist and it reverted this :shrug:

Reply:
That is so weird. 😆